templates/migration/templates/migrationservice-configmap.yaml

```diff
@@ -5,6 +5,12 @@ data:
   SPRING_DATASOURCE_URL: "jdbc:mariadb://mariadb.{{ .Release.Namespace }}:3306/migrationservice"
   KEYCLOAK_AUTH_SERVER_URL: {{ .Values.keycloakAuthServerUrl}}
   ROCKETCHAT_CONFIG_SERVER_URL: "http://rocketchat.{{ .Release.Namespace }}:3000/api"
+  {{- if .Values.global.usePublicKeycloakUrlForMigration }}
```

Comment:
do we need this feature flag at all? is it not enough to switch to the internal keycloak host name?

Reply:
changed to use always internal keycloak